docs/workforce-management-framework/topics/pro-passportauth-pointing-to-a-datasource.adoc

```diff
@@ -0,0 +1,51 @@
+[id='{context}-pro-passportauth-pointing-to-a-datasource']
```

Comment:
This is the new name for "[id='pro-working-with-passportauth-{chapter}']"

Reply:
I'm strongly against more chapters and more files that are disjoined from original content. 
This will make reviews and updating little bit harder. 
Let's keep this simple and easy to manage.

+== Procedure == Prerequisites etc. are just noise and it will make our documentation unreadable.